torch/distributed/fsdp/fully_sharded_data_parallel.py

```diff
@@ -400,6 +401,7 @@ def __init__(
         self,
         module: nn.Module,
         process_group: ProcessGroupType = None,
+        device_mesh: Optional[DeviceMesh] = None,
```

Comment:
Is this technically BC breaking for anyone passing via positional args?

Should we be concerned about that? I do agree we should / should have enforced keyword args only for the ones after `module`.

Reply:
Great catch! Ye. We wouldn't want to break the existing workflows. Moving the kwarg to the end.